p2p_wallet/Scenes/Main/Send/Input/SendInputParameters.swift

```diff
@@ -0,0 +1,27 @@
+import SolanaSwift
+import Send
+
+struct SendInputParameters {
+    let source: SendSource
+    let recipient: Recipient
+    let preChosenWallet: Wallet?
+    let preChosenAmount: Double?
+    let pushedWithoutRecipientSearchView: Bool
+    let allowSwitchingMainAmountType: Bool
+
+    init(
```

Comment:
For internal struct without any customization initializer, there is no need to write init, as it is implicitly added.

Reply:
true, removed it